.linkmllint.yaml

```diff
@@ -0,0 +1,67 @@
+# LinkML Linter Configuration for NMDC Schema
```

Comment:
Ditto previous comment about de-duplicating with PR #2717. Please add in only one PR.